src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <div>
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+
+    <p className="Person__age">{person.age ? `I am ${person.age}` : ' '}</p>
```

Comment:
The age paragraph renders an empty space when the age is not provided. Consider rendering nothing or a more meaningful message instead of a single space to avoid unnecessary empty elements in the DOM.